src/utils/mcrypto.ts

```diff
@@ -0,0 +1,9 @@
+import bcrypt from 'bcryptjs';
+
+export const comparePwd = async (pwd: string, hash: string): Promise<boolean> => {
+  return bcrypt.compare(pwd, hash);
+};
+
+export const hashPwd = async (pwd: string, salt = 10): Promise<string> => {
+  return bcrypt.hash(pwd, salt);
```

Comment:
The `salt` param here is not a real salt. Also, real salt should be persisted to database.

https://github.com/dcodeIO/bcrypt.js/blob/master/src/bcrypt.js#L164